### ISSUE TITLE
Crash trying to create a more complex layout (UICollectionReusableView with a UICollectionView inside)

### ISSUE THREAD

**Issue by jyounus** [jamztang/CSStickyHeaderFlowLayout#121]
Hey there,

Thanks for creating this and making this open source. 
I've got a UICollectionView with this flow layout, and it works fine. 

However, now when I try to make the "CSStickyHeaderParallaxHeader" supplementary view be a UICollectionReusableView but with a UICollectionView inside it, it crashes for me when I scroll up.

The design I'm trying to implement is for browsing profiles. I want the CSStickyHeaderParallaxHeader supplementary view to show a list of images the user can swipe left/right. As soon as the user swipes up/down, the supplementary view should stay and not move (like your Spotify example). 

However because I'm using a UICollectionReusableView with a UICollectionView inside, it seems to crash as soon as I swipe up. It gives me this error in the console:

```
2016-01-05 21:31:58.048 App[22890:3720901] the behavior of the UICollectionViewFlowLayout is not defined because:
2016-01-05 21:31:58.049 App[22890:3720901] the item height must be less than the height of the UICollectionView minus the section insets top and bottom values, minus the content insets top and bottom values.
2016-01-05 21:31:58.049 App[22890:3720901] The relevant UICollectionViewFlowLayout instance is <CoverflowFlowLayout: 0x7fa7db5d3610>, and it is attached to <UICollectionView: 0x7fa7dc063400; frame = (0 0; 375 168); clipsToBounds = YES; autoresize = RM+BM; gestureRecognizers = <NSArray: 0x7fa7db5d1e70>; layer = <CALayer: 0x7fa7db5cee70>; contentOffset: {0, 0}; contentSize: {1503.3404761904762, 190}> collection view layout: <CoverflowFlowLayout: 0x7fa7db5d3610>.
2016-01-05 21:31:58.049 App[22890:3720901] Make a symbolic breakpoint at UICollectionViewFlowLayoutBreakForInvalidSizes to catch this in the debugger.
```

I can bypass this crash if I set the "parallaxHeaderMinimumReferenceSize" of your flow layout to be the same as the "parallaxHeaderReferenceSize". However now I don't get the parallax effect. The supplementary view moves up as well. It's not "sticky" in the background, if that makes sense. I want to have an "overlay" effect (again something similar to the Spotify example).

Do you have any ideas what might be causing this issue? Let me know if you need more info or if you want me to record a short video demonstrating my problem, I can send it to you via email.

Thanks

**Comment by himandhawan01** [comment]
Yes , i got the same issue ..... but i want parallax effect , did you find any solution ?
